### DISCUSSION TITLE
Generator: Jailbroken vs. Unjailbroken

### DISCUSSION THREAD

**Discussion by andreaswilli** [airsquared/blobsaver#302] [Q&A]
First of all, thanks for the amazing tool!

I'm using v3.0.1 which offers the option to read the generator from a device. However, I'm not sure I fully understand the `jailbroken` and `unjailbroken` options. Does `unjailbroken` mean the blobs will be valid regardless of whether my device is jailbroken or not? If so, what's the `jailbroken` option for? Or does it mean something else entirely?

I'd appreciate if you could help me decide which option (or both?) to use.

**Comment by r1sk01** [comment]
@andreaswilli, use the latest version as blobsaver will configure it for you, however there is a difference between jailbroken and unjailbroken, basically, jailbroken will use 0x1111111111111111 and unjailbroken uses the actual generator on the device (jailbroken devices wipe their own generator to the one I mentioned because the kernel of iOS was modified and the generator stays until you restore RFS)
